src/ensembl/production/gifts/templates/submit.html

```diff
@@ -1,17 +1,25 @@
 
 {% extends 'base.html' %}
 
-{% set title = 'GIFTs: Submit Job' %}
 {% set icon_image = 'img/gifts.png' %}
+{% set title = 'GIFTs Service' %}
```

Comment:
For the icon image, I think it's best to use absolute path.

Reply:
An absolute path is created when this variable is passed to the `url_for` method, it ends up being `/static/img/gifts.png`

Reply:
sounds about right. btw, regarding upcoming ENSPROD-7097 doing, If you have any JS associated somewhere which rely on absolute paths, please tell me, we'll discuss how to get rid of it!